examples/curations.yml

```diff
@@ -59,3 +59,9 @@
     comment: "The package has an unmappable declared license entry."
     declared_license_mapping:
       "BSD-derived (http://www.repoze.org/LICENSE.txt)": "LicenseRef-scancode-repoze"
+
+- id: "PyPI::branca"
+  curations:
+    comment: "This is a copyright statement, not a license."
+    declared_license_mapping:
+      Copyright (C) 2013, Martin Journois: "NONE"
```

Comment:
I'm surprised that we don't need quotes for the key here, like in line 61. So, it seems keys with spaces do not need quoting, but keys with other special characters (like the colon as part of the URL in line 61) do?

Reply:
How about consistently using quotes for keys and values of `declared_license_mapping` ?

Reply:
Added the quotes.